src/main/resources/assets/commandblockide/lang/de_de.json

```diff
@@ -0,0 +1,23 @@
+{
+    "arguments.editfunction.tag.unsupported": "Kann keine Funktions Tags bearbeiten",
+    "commands.editfunction.failed.modNotInstalled": "Die Command Block IDE muss installiert sein",
+    "commandBlockIDE.close": "Schließen",
+    "commandBlockIDE.save": "Speichern",
+    "commandBlockIDE.type.redstone": "Impuls",
+    "commandBlockIDE.type.redstoneKonditional": "Impuls, Konditional",
```

Comment:
The key has a `K` instead of a `C` in `Conditional`, which seems like a typo and won't work in-game.

Reply:
Actually, it shouldn't be `Konditional` but `Bedingt` (that's also how it's translated in vanilla)

Reply:
I meant in the translation key (the left side), which is used to find the text in the first place.

Reply:
Oh lol, I didn't see that